enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,1319 @@
+             - eipalloc-1111
+             - eipalloc-2222
+             - eipalloc-3333
+             - eipalloc-4444
+             - eipalloc-5555
+       type: AWS
+```
+3. Ingress Operator will create the default ingress controller with the service type load balancer having the service annotation `service.beta.kubernetes.io/aws-load-balancer-eip-allocations` having values of the field `eipAllocations` 
+   from the Ingress Controller CR.
+
+#### Create an Ingress Controller using Ingress Controller CR and assign the EIP allocation mentioned in the CR to the kubernetes Load Balancer service for the Ingress Controller.
+1. Cluster administrator creates an Ingress Controller:
+```yaml
+ % cat custom-eip-cr.yaml
+ apiVersion: operator.openshift.io/v1
+ kind: IngressController
+ metadata:
+   creationTimestamp: null
+   name: test
+   namespace: openshift-ingress-operator
+ spec:
+   domain: apps1.apps.misalunkeip16.devcluster.openshift.com
+   endpointPublishingStrategy:
+     loadBalancer:
+       scope: External
+       providerParameters:
+         type: AWS
+         aws:
+           type: NLB
+           networkLoadBalancer:
+             eipAllocations:
+               - eipalloc-0956fea34de4cb7ab
+               - eipalloc-0e9a3077a70de050a
+               - eipalloc-0b69fc4691f54cdd0
+               - eipalloc-01e6ba6cbba1a391b
+               - eipalloc-0e242df173f906112
+     type: LoadBalancerService
+```
+```sh
+% oc create -f custom-eip-cr.yaml
+ingresscontroller.operator.openshift.io/test created
+```
+```sh
+% oc get ingresscontrollers/test -n openshift-ingress-operator
+NAME   AGE
+test   118s
+```
+
+Ingress Operator will create an NLB Ingress Controller called `test` with the service type load balancer having the service annotation `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`
+populated with the value from the eipAllocations field.
+
+```yaml
+% oc get ingresscontroller test -n openshift-ingress-operator -o yaml
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  creationTimestamp: "2024-04-10T00:20:17Z"
+  finalizers:
+  - ingresscontroller.operator.openshift.io/finalizer-ingresscontroller
+  generation: 2
+  name: test
+  namespace: openshift-ingress-operator
+  resourceVersion: "58136"
+  uid: 5f6da5d0-4ef1-429e-ba32-af35ad260c89
+spec:
+  clientTLS:
+    clientCA:
+      name: ""
+    clientCertificatePolicy: ""
+  domain: apps1.apps.misalunkeip16.devcluster.openshift.com
+  endpointPublishingStrategy:
+    loadBalancer:
+      dnsManagementPolicy: Managed
+      providerParameters:
+        aws:
+          networkLoadBalancer:
+            eipAllocations:
+            - eipalloc-0956fea34de4cb7ab
+            - eipalloc-0e9a3077a70de050a
+            - eipalloc-0b69fc4691f54cdd0
+            - eipalloc-01e6ba6cbba1a391b
+            - eipalloc-0e242df173f906112
+          type: NLB
+        type: AWS
+      scope: External
+    type: LoadBalancerService
+  httpCompression: {}
```

Comment:
Everything between here and L303 can be replaced with `...` for clarity.

Reply:
I have updated the yaml.